### PR TITLE
add edge dependency to podspec

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

**Review by nporter-adbe — APPROVED** [review]

File: AEPEdgeConsent.podspec [hunk 1]
```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
 
   s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
   s.dependency 'AEPCore', '>= 3.1.0'
+  s.dependency 'AEPEdge', '>= 1.1.1'
```
Comment (emdobrin) [inline AEPEdgeConsent.podspec:19]: it required min 1.1.0

**Review by emdobrin — COMMENTED** [review]
Thanks! can we add it to package.swift too?

**Comment by sbenedicadb** [comment]
@emdobrin done!

File: Package.swift [hunk 2]
```diff
@@ -22,11 +22,12 @@ let package = Package(
         .library(name: "AEPEdgeConsent", targets: ["AEPEdgeConsent"])
     ],
     dependencies: [
-        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .branch("main"))
+        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .branch("main")),
```
Comment (emdobrin) [inline Package.swift:25]: let's update this one as well to .upToNextMajor(from: "3.1.0"))

**Review by emdobrin — APPROVED** [review]

File: Package.swift [hunk 3]
```diff
@@ -22,11 +22,12 @@ let package = Package(
...
-        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .branch("main"))
+        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "3.1.0")),
+        .package(url: "https://github.com/adobe/aepsdk-edge-ios.git", .upToNextMajor(from: "1.1.0"))
     ],
     targets: [
         .target(name: "AEPEdgeConsent",
-                dependencies: ["AEPCore"],
+                dependencies: ["AEPCore", "AEPEdge"],
                 path: "Sources"),
         .testTarget(name: "FunctionalTests",
```
Comment (kevinlind) [inline Package.swift:32]: Let's remove the test targets here.

**Review by kevinlind — CHANGES_REQUESTED** [review]
Can you update the README.md similar to this PR? https://github.com/adobe/aepsdk-edgeidentity-ios/pull/42

**Review by kevinlind — APPROVED** [review]

**Pull request merged**